src/app/dim-ui/RichDestinyText.tsx

```diff
@@ -147,18 +147,13 @@ const generateConversionTable = _.once((defs: D2ManifestDefinitions) => {
   });
 });
 
-const replaceWithIcon = (textSegment: string) => {
+const replaceWithIcon = (textSegment: string, index: number) => {
   const replacement = baseConversionTable.find(
     (r) => r.substring === textSegment || r.unicode === textSegment
   );
   return (
     (replacement && (
-      <img
-        src={replacement.icon}
-        className={styles.inlineSvg}
-        title={textSegment}
-        key={textSegment}
-      />
+      <img src={replacement.icon} className={styles.inlineSvg} title={textSegment} key={index} />
```

Comment:
We actually have text that has two barrier icons in it. This was causing a react key error. I just swapped it to an index as its not important rendering wise and it removes the error.